walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,214 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.TreeSet;
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
+  /**
+   * Finds the possible TimeRanges where the mandatory attendees can attend
```

Comment:
```suggestion
   * Finds the possible TimeRanges where the mandatory attendees can attend.
```
Forgot the `.` at the end of the sentence. I think some other places have it too!